GPU/Vulkan/FramebufferVulkan.cpp

```diff
@@ -583,36 +670,31 @@ void FramebufferManagerVulkan::ResizeFramebufFBO(VirtualFramebuffer *vfb, u16 w,
 	}
 
 	textureCache_->ForgetLastTexture();
-
 	if (!useBufferedRendering_) {
 		if (vfb->fbo_vk) {
 			delete vfb->fbo_vk;
-			vfb->fbo_vk = 0;
+			vfb->fbo_vk = nullptr;
 		}
 		return;
 	}
 
 	vfb->fbo_vk = new VulkanFBO();
-	// bo_create(vfb->renderWidth, vfb->renderHeight, 1, true, (FBOColorDepth)vfb->colorDepth);
+	vfb->fbo_vk->Create(vulkan_, rpLoadColorLoadDepth_, vfb->renderWidth, vfb->renderHeight, framebufFormat);
 	if (old.fbo_vk) {
 		INFO_LOG(SCEGE, "Resizing FBO for %08x : %i x %i x %i", vfb->fb_address, w, h, vfb->format);
 		if (vfb->fbo_vk) {
 			/// fbo_bind_as_render_target(vfb->fbo_vk);
-			ClearBuffer();
-			if (!g_Config.bDisableSlowFramebufEffects) {
-				BlitFramebuffer(vfb, 0, 0, &old, 0, 0, std::min(vfb->bufferWidth, vfb->width), std::min(vfb->height, vfb->bufferHeight), 0);
-			}
+			//ClearBuffer();
```

Comment:
Maybe the smarter thing to do here is to set some state on the vfb, telling it to blit from the old image as soon as a render or texture fetch starts?  Then it can combine any clears/etc.  That may be more efficient and skip some blits if possible.

-[Unknown]